CRM/Core/Form.php

```diff
@@ -1254,7 +1254,12 @@ public function addField($name, $props = array(), $required = FALSE) {
       //case 'Select Date':
       //TODO: Add date formats
       //TODO: Add javascript template for dates.
-      // case 'Radio':
+      case 'Radio':
+        $separator = isset($props['separator']) ? $props['separator'] : NULL;
+        unset($props['seperator']);
```

Comment:
Proposed addition:

    // Default to allowClear if not specified
    if (!isset($props['allowClear'])) {
      $props['allowClear'] = !$required;
    }